dandiapi/api/management/commands/import_dandisets.py

```diff
@@ -1,14 +1,65 @@
 from urllib.parse import urljoin
+from uuid import uuid4
 
+from django.core.files.uploadedfile import SimpleUploadedFile
 from django.db import transaction
 import djclick as click
 import requests
 
-from dandiapi.api.models import Dandiset, Version
+from dandiapi.api.models import Asset, AssetBlob, Dandiset, Version, ZarrArchive
 
 
 @transaction.atomic
-def import_versions_from_response(api_url: str, version_api_response: dict, dandiset: Dandiset):
+def import_assets_from_response(api_url: str, asset_api_response: dict, version: Version):
+    uploaded_file = SimpleUploadedFile(name='foo/bar.txt', content=b'A' * 20)
+    etag = '76d36e98f312e98ff908c8c82c8dd623-0'
+    asset_blob = AssetBlob.objects.get_or_create(
+        etag=etag, defaults=dict(blob_id=uuid4(), blob=uploaded_file, etag=etag, size=20)
+    )[0]
+
+    for result in asset_api_response['results']:
+        blob = result.get('blob')
+        zarr = result.get('zarr')
+
+        if blob:
+            asset_metadata = requests.get(
+                urljoin(
+                    api_url,
+                    f'/api/dandisets/{version.dandiset.identifier}/versions/'
+                    f'{version.version}/assets/{result["asset_id"]}/',
+                ),
+                {'page_size': 1000},
+            ).json()
+            asset = Asset.objects.create(
+                blob=asset_blob, metadata=asset_metadata, path=result['path']
+            )
+            click.echo(f'    Importing AssetBlob "{asset.path}"')
+            version.assets.add(asset)
+
+        elif zarr:
+            zarr_info = requests.get(urljoin(api_url, f'/api/zarr/{zarr}/')).json()
+            zarr_archive = ZarrArchive.objects.create(
+                name=zarr_info['name'],
+                dandiset=version.dandiset,
+                zarr_id=zarr_info['zarr_id'],
+                file_count=zarr_info['file_count'],
+                size=zarr_info['size'],
+                status=zarr_info['status'],
+            )
+            asset = Asset.objects.create(zarr=zarr_archive, path=zarr_info['name'])
+            click.echo(f'    Importing ZarrArchive "{asset.path}"')
+            version.assets.add(asset)
+
+    # Handle API pagination
+    if asset_api_response.get('next'):
+        next_assets = requests.get(asset_api_response.get('next')).json()
+        import_assets_from_response(api_url, next_assets, version)
```

Comment:
If a version has lots of assets, this could recurse considerably, potentially leading to higher than desired memory usage. I'd think doing something like the following would be sufficient:

```python
results = asset_api_response['results']
while results:
    for result in results:
        ...

    # Pagination
    next_link = results.get('next')
    if not next_link:
        break

    results = requests.get(next_link).json()['results']
```